docs/evm/truffle.md

```diff
@@ -0,0 +1,143 @@
+---
+id: truffle
+title: Using Truffle with the EVM
+sidebar_label: Using Truffle
+---
+
+[Truffle](https://trufflesuite.com) is a popular tool for EVM-based smart contract development and testing. A typical dApp using Truffle will have a file called `truffle-config.js` in the project root specifying details on connection, compilation, networks, and so on.
```

Comment:
The actual tool is at https://www.trufflesuite.com/truffle, the whole tool suite at https://www.trufflesuite.com.